tron/utils/scribereader.py

```diff
@@ -186,6 +199,14 @@ def read_log_stream_for_action_run(
     # XXX: for some reason, we're occasionally getting data out of order from scribereader - so we'll sort based on
     # timestamp until we can figure out what's causing this.
     output.sort(key=operator.itemgetter(0))
-    return [line for _, line in output] + (
-        [f"{malformed_lines} encountered while retrieving logs"] if malformed_lines else []
+    return (
+        [line for _, line in output]
+        + ([f"{malformed_lines} encountered while retrieving logs"] if malformed_lines else [])
+        + (
+            [
+                f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} -f stream_paasta_app_output_{namespace}_{job_name}__{action} --min-date {min_date} --max-date {max_date}'"
```

Comment:
nit:
```suggestion
                f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} {stream_name} --min-date {min_date} --max-date {max_date}'"
```

also: max_date can be None, so we'll need to account for that and only include that arg if it's present - i've also noticed that i might have messed up when i was initially writing this code: can you verify that min_date and max_date here are printed out in YYYY-MM-DD format and not a full datetime string?

Reply:
I think it is datetime string. Below is how it's outputted:

```
"This output is truncated. Use this command to view all lines 'scribereader -s infrastage stream_paasta_app_output_namespace_job__action --min-date 2022-08-13 15:50:10.623651 --max-date 2022-08-14 15:50:10.623656'"]
```

Reply:
ah, we'll probably wanna call `.date()` on those to get just the YYYY-MM-DD bits